tests/components/zha/test_discover.py

```diff
@@ -380,23 +386,26 @@ async def test_device_override(hass, zigpy_device_mock, setup_zha, override, ent
         "00:11:22:33:44:55:66:77",
         "manufacturer",
         "model",
+        patch_cluster=False,
     )
 
     if override is not None:
         override = {"device_config": {"00:11:22:33:44:55:66:77-1": {"type": override}}}
 
     await setup_zha(override)
-    assert hass.states.get(entity_id) is None
-    zha_gateway = get_zha_gateway(hass)
+    assert hass_disable_services.states.get(entity_id) is None
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5801702)